### DISCUSSION TITLE
Error: WebSocket connection closed abnormally with code: 3000 (Authorization error: Unauthorized: origin not allowed)

### DISCUSSION THREAD

**Discussion by siosio34** [WalletConnect/walletconnect-monorepo#2834] [Developer Support] [answered]
Hello, I am reporting this error while using Wallet Connect.

There is an issue where the wallet connection using Wallet Connect does not work properly.

The error is popping up the same as the title.

The project ID of our WALLETCONNECT EXPLOERE is as follows.

2605733261bd9dc1fb816e17bc6c9ed7

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/7166022/00e02a84-20ed-4f59-a795-f7ac3c8c01cc)

**Comment by finessevanes** [comment]
@siosio34, this error usually occurs when there is a misconfiguration in the domain settings within the cloud. Could you please provide details on what you have in your domain settings in the cloud?

**Reply by siosio34** [reply]
@finessevanes 

my cloud id is below
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/7166022/df9f0176-4408-4b63-bda8-f0072850339d)

**Reply by siosio34** [reply]
My personal IP address has been covered. The website address of the actual service is as follows.

https://upbit.com/nft/

**Reply by siosio34** [reply, marked as answer]
It was my mistake. It was solved by adding https://upbit.com to allowed domains.
